audio/audacity/Portfile

```diff
@@ -6,6 +6,14 @@ PortGroup           active_variants 1.1
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           github 1.0
 PortGroup           wxWidgets 1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           active_variants 1.1
+
+platform darwin {
+    if {${os.major} < 13} {
+        PortGroup   cxx11 1.1
```

Comment:
(Do you want to say that in order to compile this on linux you don't need C++11?)

You should not make this conditional. Just put the portgroup on top next to all others, no conditionals.